SearchParameters/SearchParameter-author.xml

```diff
@@ -0,0 +1,22 @@
+<SearchParameter xmlns="http://hl7.org/fhir">
+  <id value="author"/>
+  <url value="http://nictiz.nl/fhir/SearchParameter/author"/>

```

Comment:
This `url` is too generic, which will result in problems when another information standard wants to adopt a SearchParameter with the same `url`). There are two solutions:
- Add it to the nl-core package, so that it can be applied to other resources as well in the future (but then a new nl-core package is necessary, and the nl-core dependency has to be updated within the mp package).
- Change the `url` to e.g. _http://nictiz.nl/fhir/SearchParameter/MedicationStatement-author_

See section 12.1.1.6 onhttps://informatiestandaarden.nictiz.nl/wiki/FHIR:V1.0_FHIR_Profiling_Guidelines_R4#Canonical_URL.2C_id.2C_name_and_title.

Reply:
This comment applies to some of the other SearchParameters as well, e.g. location and reason.

Reply:
Changing the `url` won't be done for the beta.4 release. Made a Jira ticket to fix it in the next release: https://nictiz.atlassian.net/browse/NSM-1268